### DISCUSSION TITLE
Is there a way to use OwnTracks as "source" and have up to date data from there?

### DISCUSSION THREAD

**Discussion by ericukk** [Freika/dawarich#571] [Q&A] [answered]
I'm still sending data to owntracks instead of directly to dawarich, since I'm using the sharing feature from there.
However I'd also like to be able to use dawarich as frontend app.
Any suggestion to have this set up? Maybe automatic import every hour?

**Comment by ajpanton** [comment]
https://dawarich.app/docs/tutorials/track-your-location

I'd guess what you're asking is one of the most common ways to do it. However I don't think it's possible to "import every hour". The owntracks app sends its data when it wants to, and dawarich just receives it. Though I'm not sure if I understood you correctly.

**Reply by ericukk** [reply]
actually I'm sending the data to owntracks's backend database.

**Reply by ericegonz** [reply]
I took a bit of a different approach and stood up a simple Docker service that subscribes to my Mosquitto instance for OwnTracks and forwards data into the Dawarich API. 

https://github.com/ericegonz/owntracks-dawarich-bridge

**Comment by Freika** [comment, marked as answer]
I think you could hack a script to copy your .rec file from owntracks directory to the one in Dawarich using [Watcher](https://dawarich.app/docs/tutorials/watcher) functionality

**Reply by ericukk** [reply]
Thanks. I tried that, but it failed to import. Since the documentation mentions GPX and GeoJSON, I assumed the issue was with the format. I'll give it another shot.

**Reply by Freika** [reply]
> Thanks. I tried that, but it failed to import. Since the documentation mentions GPX and GeoJSON, I assumed the issue was with the format. I'll give it another shot.

Ah sorry, it's not released yet, my mistake. But it will be soon!

**Reply by ericukk** [reply]
Many thanks for letting me know. That's very fast development. 🚢

**Reply by Freika** [reply]
@ericukk in one of the recent releases support for importing .rec files via Watcher was implemented, you can try it out now!

**Reply by ericukk** [reply]
:rocket: thanks, will check it out!

**Reply by ericegonz** [reply]
In my experience OwnTracks uses a single file per month, appending as more data is ingested. Because the Watcher ignores filenames after an import, this may not be ideal
